lib/common-crypto-fallback.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/*eslint-env browser*/
+
+// Browser adapter for required functions from node.js crypto module.
+let crypto;
+if (window.crypto && window.crypto.getRandomValues) {
+  crypto = {
+    randomBytes(count) {
+      const buf = Buffer.alloc(count);
+      window.crypto.getRandomValues(buf);
+      return buf;
+    },
+
+    randomFillSync(buf) {
+      window.crypto.getRandomValues(buf);
+    },
+  };
+} else {
+  console.warn(
+    'Web Crypto API is not supported in your browser!',
+    'Using Math.random() instead.'
+  );
+  const getRandomByte = () => Math.floor(0x100 * Math.random());
+  crypto = {
+    randomBytes(count) {
+      const buf = Buffer.alloc(count);
+      for (let i = 0; i < count; i++) {
```

Comment:
Not that essential as it's only three lines of code, but the function below could be reused here.